Server/src/main/resources/config.properties

```diff
@@ -4,9 +4,14 @@
 #MacOSX: /Users/yourusername/Library/Application Support/cardboard/1.0
 #Windows: C:\Users\yourusername\AppData\Roaming\projectcardboard\cardboard\1.0
 #Linux: /home/yourusername/.config/cardboard/1.0
-HOST_SERVER = false
-SERVER_IP = 3.123.26.96
-SERVER_URI = wss://mechaz.org
-PORT = 8888
-SERVER_NAME = Server
-API_HOST = mechaz.org
\ No newline at end of file
+HOST_SERVER=false
+SERVER_IP=3.123.26.96
+SERVER_URI=wss://mechaz.org
+PORT=8888
+SERVER_NAME=Server
+API_HOST=mechaz.org
+
+#UI Related Variables
+SCREEN_WIDTH=1920
+SCREEN_HEIGHT=1080
+FULLSCREEN=true 
```

Comment:
Did your autoformatter switch this file to no spaces between identifiers?

Reply:
in #493 we have a method that appends to the user's config file the missing config variables if theres any. But the check to see if the user is missing some variables only works if we remove the spaces between identifiers.